sdk/lib/atl/atldef.h

```diff
@@ -7,6 +7,9 @@
 
 #pragma once
 
+#ifndef __REACTOS__
+    #include <pseh/pseh2.h>
+#endif
```

Comment:
Why is this needed?
`PSEH` is a reactos thing.

Reply:
Now `RATL` has `PSEH`.